src/content/components/uikit/UserLabel/index.ts

```diff
@@ -19,7 +20,7 @@ export const userLabelConfig = {
             avatar: {
                 type: 'input',
                 defaultValue:
-                    'https://upload.wikimedia.org/wikipedia/commons/thumb/3/33/Charles_Darwin_by_Julia_Margaret_Cameron%2C_c._1868.jpg/193px-Charles_Darwin_by_Julia_Margaret_Cameron%2C_c._1868.jpg',
+                  'https://upload.wikimedia.org/wikipedia/commons/thumb/3/33/Charles_Darwin_by_Julia_Margaret_Cameron%2C_c._1868.jpg/193px-Charles_Darwin_by_Julia_Margaret_Cameron%2C_c._1868.jpg',
```

Comment:
Lint error

Reply:
it was in outdated commit, it is no longer there